moveit_core/CMakeLists.txt

```diff
@@ -20,17 +20,17 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules")
 
 # TODO(henningkayser): enable bullet once the library is migrated to ROS2
 # TODO: Move collision detection into separate packages
-# find_package(Bullet 2.87)
-#
-# # TODO(j-petit): Version check can be dropped when Xenial reaches end-of-life
-# if(BULLET_FOUND)
-#   set(BULLET_ENABLE "BULLET")
-#   set(BULLET_LIB "moveit_collision_detection_bullet")
-#   set(BULLET_INC "collision_detection_bullet/include")
-#   message(STATUS "Compiling with Bullet")
-# else()
-#   message(STATUS "Version of Bullet too old or not available: disabling Bullet collision detection plugin. Try using Ubuntu 18.04 or later.")
-# endif()
+ find_package(Bullet 2.87)
```

Comment:
I think in our meeting we agreed to leave it in this version (2.87).  However let me know if you'd prefer I change this.

Reply:
Actually, I think we can ignore the case that bullet is not available because MoveIt 2 doesn't support systems older than 20.04 and even with 18.04 bullet is available with the right version. Also, rosdep already installs bullet on the system since it's listed in the package.xml. I would simply add a REQUIRED to the `find_package()` and remove the `BULLET_FOUND` check altogether.